src/web/src/components/DynamicImage.tsx

```diff
@@ -1,27 +1,41 @@
 import { makeStyles } from '@material-ui/core/styles';
 
+import { imageServiceUrl } from '../config';
+
 const useStyles = makeStyles((theme) => ({
   background: {
-    backgroundPosition: 'center',
-    backgroundRepeat: 'no-repeat',
-    backgroundAttachment: 'scroll',
-    backgroundSize: 'cover',
-    transition: 'opacity 1s ease-in-out',
-    height: 'calc(100vh - 64px)',
-    [theme.breakpoints.between('xs', 'sm')]: {
-      height: 'calc(100vh - 56px)',
+    display: 'block',
+    height: '100%',
+    // TODO: this isn't right...
+    [theme.breakpoints.down(1024)]: {
+      height: 'calc(100vh - 64px)',
```

Comment:
The way you do breakpoints here is legit, since MUI does not have any generic breakpoint at 1024px. Here is the chart of MUI's breakpoints
![image](https://user-images.githubusercontent.com/55034244/111053895-ccdb7680-8435-11eb-92d6-b6418843612c.png)
About having a more dynamic height for the bottom bar, I think we can use ref to refer to the `NavBar` component, something like this 
![image](https://user-images.githubusercontent.com/55034244/111054034-242e1680-8437-11eb-82ef-e2757e8c6dfa.png)
And then we can having a context to provide the height using `navBarRef.current.offsetHeight` to the `<Component />` in index page. Also, we could have a hook to use it and update a state in `<DynamicImage />` so it can rerender if there is a change in height of NavBar.
Reference: https://stackoverflow.com/questions/50747953/how-to-get-the-height-of-a-react-element